packages/integrations/src/plex/plex-integration.ts

```diff
@@ -0,0 +1,102 @@
+import { parseStringPromise } from "xml2js";
+
+import { logger } from "@homarr/log";
+
+import { Integration } from "../base/integration";
+import { IntegrationTestConnectionError } from "../base/test-connection-error";
+import type { StreamSession } from "../interfaces/media-server/session";
+import type { PlexResponse } from "./interface";
+
+export class PlexIntegration extends Integration {
+  public async getCurrentSessionsAsync(): Promise<StreamSession[]> {
+    const token = super.getSecretValue("apiKey");
+
+    const response = await fetch(`${this.integration.url}/status/sessions`, {
+      headers: {
+        "X-Plex-Token": token,
+      },
+    });
+    const body = await response.text();
+    // convert xml response to objects, as there is no JSON api
+    const data = await PlexIntegration.parseXml<PlexResponse>(body);
+    const mediaContainer = data.MediaContainer;
```

Comment:
Funny thing, oldmarr used element[0], meaning it didn't actually target a "MediaContainer".
In the case of music, the element is actually a "Track" and not a "MediaContainer".
Oldmarr was working by mistake. In your case though, you need to handle "data.Track" instead for music.